samples/nrf9160/https_client/src/main.c

```diff
@@ -167,20 +168,25 @@ void main(void)
 		printk("Failed to connect to the LTE network, err %d\n", err);
 		return;
 	}
-	printk("OK\n");
 
-	err = getaddrinfo(HTTPS_HOSTNAME, NULL, &hints, &res);
+	snprintf(portstr, 6, "%d", HTTPS_PORT);
```

Comment:
Why do you need to format the port number in a dedicated buffer?
I reckon it should be sufficient to change the format you pass to `printk` to let it print an integer.

Reply:
It was more because the value passed into getaddrinfo is a string (rather than set it afterwards); but often a passed in config or stored value will be a string anyway. I changed the define to be a string from the start.